### PR TITLE
Math magicians: fetch data from API

### PR DESCRIPTION
**Pull request by Titus-Kiplagat**
- [x] Used the following external API: [API Ninjas](https://api-ninjas.com/api/quotes).
- [x] Created a new component `QuoteDisplay.jsx` for displaying a quote.
- [x] Fetched data inside of a new component.
- [x] Show a `loading` state.
- [x] Show an `error` state.
- [x] Used the correct `useEffect` React hook when fetching data.
- [x] Connected to an API within a functional component
- [x] Show different UI outputs depending on the component state

### CODE REVIEW

File: src/components/QuoteDisplay.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        {' '}
+        by
+        {' '}
+        <span>
+          &#34;
+          {author}
+          &#34;
+        </span>
+      </h3>
+    </div>
```
Comment (thecodechaser) [inline src/components/QuoteDisplay.jsx]: - [ ] I can't see quotes displayed on UI there's a JSON error, Tried many times. Please make sure to fix this issue and check the API is working as expected.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Titus-Kiplagat👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OLIPLICHE** [comment]
@Titus-Kiplagat can u stop pushing? your app is under review. you creating confusion

**Comment by Titus-Kiplagat** [comment]
> @Titus-Kiplagat can u stop pushing? your app is under review. you creating confusion

Yes, Sir.

**Review by OLIPLICHE — APPROVED** [review]
Hi @Titus-Kiplagat, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Quotes component has been created (✅)) 👏
- The external `API` has been used correctly👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Titus-Kiplagat** [comment]
@OLIPLICHE 
I've got a question. I tried working with environment variables in this react project as you can see the variables declared in the `.env.example` file. One of the variables is  `ninjas' base URL`. The issue occurs when trying to access those variables in `components`.
For my instances accessing the `base URL` for the fetch method returns undefined. Any suggestions on how to use environment variables in react applications?

**Pull request merged**